qchem/CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 <h3>New features since last release</h3>
 
+* The ``derivative()`` and ``gradient()`` functions have been added to compute
+  the gradient of molecular Hamiltonians with respect to the nuclear
+  coordinates using a finite-difference approximation.
+
+  For example:
+
+  ```pycon
+  >>> def H(x):
```

Comment:
This is a minor point but we might use ``r`` for the coordinates vector.

Reply:
Yes. The problem is that might be confusing as the this is standard notation for the electron coordinates.